corehq/apps/reports/analytics/esaccessors.py

```diff
@@ -67,6 +68,21 @@ def convert_to_date(date):
     return result
 
 
+def get_days_since_last_form_submission_for_user_id(domain, user_id):
+    query = (
+        FormES()
+        .domain(domain)
+        .user_id(user_id)
+        .sort('received_on', desc=True)
+        .size(1)
+    )
+    result = query.run().hits
+    if not result:
+        return -1
+    delta = datetime.utcnow() - iso_string_to_datetime(result[0]['received_on'])
```

Comment:
Is there any chance that `result` may contain data that causes errors here? For example, if `received_on` has a bad format or isn't a string. Those scenarios appear to be unhandled and untested.

Reply:
is that ever the case in elasticsearch that received on would be anything other than a date? Then I think we would have bigger problems with our indexing, no?

Reply:
this method was removed, so closing out thread :)